application/holder/src/service/updater/releasefile.ts

```diff
@@ -14,12 +16,15 @@ const PATERN = `${HOOKS.name}${HOOKS.prefix}@${HOOKS.version}-${HOOKS.platform}-
 
 export class ReleaseFile {
     public readonly filename: string;
-
-    constructor(version: string, prefix = '') {
-        const platform: Platform = getPlatform();
-        if (platform === Platform.undefined) {
-            throw new Error(`Fail to detect supported platform for (${os.platform()}).`);
+    constructor(version: string, prefix = '', platform?: string) {
```

Comment:
As soon as `prefix` now isn't last argument any more, let's make it `requered`:
```
constructor(version: string, prefix: string, platform?: string) { }
```
I will not break any calls in other places.